server/src/main/java/org/opensearch/search/profile/AbstractProfileBreakdown.java

```diff
@@ -50,8 +50,8 @@ public abstract class AbstractProfileBreakdown<T extends Enum<T>> {
     /**
      * The accumulated timings for this query node
      */
-    private final Timer[] timings;
-    private final T[] timingTypes;
+    public final Timer[] timings;
+    public final T[] timingTypes;
```

Comment:
I think we can keep these `private`. Don't see why `buildBreakdownMap` was implemented to take in an instance of `AbstractProfileBreakdown`. We should be able to remove that method and instead put the logic in the `toBreakdownMap` and use that everywhere

Reply:
The matter has been issued now.